frontend/package-lock.json

```diff
@@ -1,11 +1,11 @@
 {
   "name": "frontend",
-  "version": "1.0.0-16.19",
+  "version": "1.0.0-18.1",
```

Comment:
Please rebase your branch. This would cause conflicts if merged as-is. DEV is currently at IS19

Reply:
I just checked dev and it has "1.0.0-16.19" as the version, but it should be 18.1 according to the packge.json file. Note that it is generated file from the package.json one

Reply:
DEV is showing 19.3 for me. Maybe you have an old version cached in your browser?
![image](https://user-images.githubusercontent.com/24322224/149214164-1f67424e-542a-4a12-b5df-e52e6320eca7.png)

Reply:
Also **package.json** is showing the correct value in git
![image](https://user-images.githubusercontent.com/24322224/149214280-17b14f28-f383-49c0-94fa-f350b3503b34.png)

Reply:
I think there is confusion, the comment is about package-lock.json. not package.json

Reply:
Oh I see. Sorry, missed that. Disregard my previous comments. It seems the lock file gets updated only when someone does `npm install` locally. 

You are right - we don't care about the value in the lock file